src/backend/optimizer/plan/aqumv.c

```diff
@@ -398,7 +410,6 @@ answer_query_using_materialized_views(PlannerInfo *root,
 		 * We don't use STD_FUZZ_FACTOR for cost comparisons like compare_path_costs_fuzzily here.
 		 * The STD_FUZZ_FACTOR is used to reduce paths of a rel, and keep the significantly ones.
 		 * But in AQUMV, we always have only one best path of rel at the last to compare.
-		 * TODO: limit clause and startup_cost.
 		 */
 		if (mv_final_rel->cheapest_total_path->total_cost < current_rel->cheapest_total_path->total_cost)
```

Comment:
has limit , use cheapest_startup_path to compare cost ?

Reply:
Reason?

Reply:
best_path is always selected from cheapest_total_path, cheapest_startup_path is considered for join, it's not related to LIMIT.